core/validation_schema.py

```diff
@@ -26,6 +26,7 @@
             "Organisation": "str",
         },
         "uniques": ["Programme ID"],
+        "composite_keys": ("Programme ID", "Organisation"),
```

Comment:
I think if the implementation expects a single composite key like this then the name of this attribute should be singular i.e. "composite_key" rather than "composite_keys"

Reply:
same for the rest of the instances of this in this file, the code that references it, and the tests